components/app/Calendar.vue

```diff
@@ -241,10 +245,18 @@ const groupedEvents = computed(() => {
         />
         <div>
           <strong>Event details:</strong>
-          <ul>
+          <ul class="mb-4">
             <li>Event starts at: {{ selectedEvent.start.formatTime(TIME_FORMAT) }}</li>
             <li>Event ends at: {{ selectedEvent.end.formatTime(TIME_FORMAT) }}</li>
           </ul>
+
+          <div>
+            <NuxtLink
+              :href="selectedEvent.eventUrl"
+              target="_blank"
```

Comment:
suggestion: use `NuxtLink` native props:
```suggestion
              :to="selectedEvent.eventUrl"
              external
```